File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.21.java
// Dado três números digitados pelo usuário, e todos diferentes, imprima o menor número
import java.util.Scanner;

public class Exercicio21 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o número 1: ");
        int n1 = scanner.nextInt();

        System.out.print("Digite o número 2: ");
        int n2 = scanner.nextInt();

        System.out.print("Digite o número 3: ");
        int n3 = scanner.nextInt();

        if (n1 == n2 || n1 == n3 || n2 == n3) {
            System.out.println("Os números são iguais. Pare de palhaçada.");
        } else {
            int menorNum = Math.min(n1, Math.min(n2, n3));

            System.out.println("O menor número é " + menorNum);
        }

        scanner.close();
    }
}

File: Livro - Módulo Python/3 - Estrutura de Decisão/Exercicio.21.py
#21. Dado três números digitados pelo usuário, e todos diferentes, imprima o menor número.

numeroA = float(input('Digite número A: '))
numeroB = float(input('Digite número B: '))
numeroC = float(input('Digite número C: '))

if numeroA < numeroB and numeroA < numeroC:
 print('Numero A é menor')
else:
  if numeroB < numeroC and numeroB < numeroA:
    print('Numero B é menor')
  else:
    if numeroC < numeroA and numeroC < numeroB:
      print('Numero C é menor')
    else:
      print('Eles são iguais')